.github/workflows/ci.yml

```diff
@@ -40,17 +40,17 @@ jobs:
         libsqlite3-mod-spatialite \
         gdal-bin
 
-    - name: Install python dependencies
-      run: pip install -r requirements-test.txt
-
```

Comment:
I would avoid moving this and pin the version also here: https://github.com/openwisp/django-loci/blob/master/requirements-test.txt#L7